HW_9/index.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+class Github {
+  constructor(client, secret) {
+    this.clientId = client;
+    this.clientSecret = secret;
+  }
+
+  async getUser(userName) {
+    const data = await fetch(`https://api.github.com/users/${userName}?client_id=${this.clientId}&client_secret=${this.clientSecret}`);
+    const parsedData = await data.json();
+    return parsedData;
+  }
+
+  async getRepos(userName) {
+    const repoData = await fetch(`https://api.github.com/users/${userName}/repos?client_id=${this.clientId}&client_secret=${this.clientSecret}&per_page=5`);
+    const parsedRepoData = await repoData.json();
+    return parsedRepoData;
```

Comment:
```suggestion
    return await repoData.json();
```
parsedRepoData - лишня змінна